Makefile

```diff
@@ -19,7 +19,7 @@ build: hardcoded/hardcoded.go
 test: build generate $(PKGS) js-tests
 
 js-tests:
-	cd test/js && npm install && npm test
+	cd test/js && rm -rf node_modules && npm install && npm test
```

Comment:
this might slow things down when developing locally--is it absolutely necessary?